README.md

```diff
@@ -2,8 +2,11 @@
 
 https://polymc.org
 
-## local development 
 
-`npm install`
+## Serve site for local development
 
-`npx @11ty/eleventy --serve`
+1. `npm install`
+
+2. `npm run serve`
+
+3. Profit!
```

Comment:
This is not required.

Reply:
Is this okay?
I mean strictly saying the other steps aren't necessary in a way either.